packages/components/src/internal/components/entities/EntityInsertPanel.tsx

```diff
@@ -871,13 +886,13 @@ export class EntityInsertPanelImpl extends Component<Props, StateProps> {
     };
 
     getInsertColumns = (): List<QueryColumn> => {
-        const model = this.getQueryGridModel();
-        let columns: List<QueryColumn> = model
+        const { dataModel } = this.state;
```

Comment:
nit: This could be `const { queryInfo } = this.state.dataModel`

Reply:
done